components/request_otr/browser/request_otr_service.h

```diff
@@ -43,11 +45,20 @@ class RequestOTRService : public KeyedService,
   ~RequestOTRService() override;
   void OnRulesReady(const std::string&) override;
 
-  bool ShouldBlock(const GURL& url) const;
+  bool ShouldOfferOTR(const GURL& url);
```

Comment:
name changes as discussed and please add some comments here and/or above in the main class comments with an  overview of the process for offer, etc...

Reply:
changed in https://github.com/brave/brave-core/commit/c83ae23838b8132c40d9ce007c338abf52804ae9